Buildscripts/CDN/upload-app-files.py

```
@@ -0,0 +1,65 @@
+    )
+    files_to_upload = os.listdir(path)
+    counter = 1
+    total = len(files_to_upload)
+    for file_name in files_to_upload:
+        if not index_only or file_name == 'apps.json':
+            object_path = f"apps/{sdk_version}/{file_name}"
+            print(f"[{counter}/{total}] Uploading {file_name} to {object_path}")
+            file_path = os.path.join(path, file_name)
+            try:
+                s3.upload_file(file_path, "tactility", object_path)
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)